src/control/controlobjectscript.cpp

```diff
@@ -56,12 +62,17 @@ bool ControlObjectScript::removeScriptConnection(const ScriptConnection& conn) {
         // no ScriptConnections left, so disconnect signals
         disconnect(m_pControl.data(),
                 &ControlDoublePrivate::valueChanged,
+                conn.proxy,
+                &CompressingProxy::slotValueChanged);
+        disconnect(conn.proxy,
+                &CompressingProxy::signalValueChanged,
                 this,
                 &ControlObjectScript::slotValueChanged);
         disconnect(this,
                 &ControlObjectScript::trigger,
                 this,
                 &ControlObjectScript::slotValueChanged);
+        delete (conn.proxy);
```

Comment:
This leaves a dangling pointer.

Reply:
...and the parentheses are uncommon and not needed.

Reply:
Removed